src/app/design/styles/typography/ux-styles-typography.component.html

```diff
@@ -0,0 +1,454 @@
+        <div class="definition-item">
+          <div class="sky-font-display-1">Display 1</div>
+        </div>
+        <div class="definition-props">
+          <div class="definition-code">
+            <div class="sky-field-label">Code</div>
+            <sky-code>&lt;h1 class=&quot;sky-font-display-1&quot;&gt;Display 1&lt;/h1&gt;</sky-code>
+          </div>
+          <div class="definition-styling">
+            <div class="sky-field-label">Styling</div>
+            <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">Blackbaud Sans</a> 48px #212327</div>
+          </div>
+        </div>
+      </div>
+      <div class="definition-table definition-table-wide">
+        <div class="definition-item">
+          <div class="sky-font-display-2">Display 2</div>
+        </div>
+        <div class="definition-props">
+          <div class="definition-code">
+            <div class="sky-field-label">Code</div>
+            <sky-code>&lt;div class=&quot;sky-font-display-2&quot;&gt;Display 2&lt;/div&gt;</sky-code>
+          </div>
+          <div class="definition-styling">
+            <div class="sky-field-label">Styling</div>
+            <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">Blackbaud Sans</a> 28px #212327</div>
+          </div>
+        </div>
+      </div>
+      <div class="definition-table definition-table-wide">
+        <div class="definition-item">
+          <div class="sky-font-display-3">Display 3</div>
+        </div>
+        <div class="definition-props">
+          <div class="definition-code">
+            <div class="sky-field-label">Code</div>
+            <sky-code>&lt;div class=&quot;sky-font-display-3&quot;&gt;Display 3&lt;/div&gt;</sky-code>
+          </div>
+          <div class="definition-styling">
+            <div class="sky-field-label">Styling</div>
+            <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">Blackbaud Sans</a> 20px #212327</div>
+          </div>
+        </div>
+      </div>
+      <div class="definition-table definition-table-wide">
+        <div class="definition-item">
+          <div class="sky-font-display-4">Display 4</div>
+        </div>
+        <div class="definition-props">
+          <div class="definition-code">
+            <div class="sky-field-label">Code</div>
+            <sky-code>&lt;div class=&quot;sky-font-display-4&quot;&gt;Display 4&lt;/div&gt;</sky-code>
+          </div>
+          <div class="definition-styling">
+            <div class="sky-field-label">Styling</div>
+            <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans-bold.woff">Blackbaud Sans Bold</a> 16px #212327</div>
+            <div>Uppercase</div>
+          </div>
+        </div>
+      </div>
+    </div> 
+
+    <div class="sky-margin-stacked-content-section">
+      <h3 class="sky-font-heading-3">Miscellaneous</h3>
```

Comment:
We don't provide guidance for usage here, but we do call out when to use de-emphasized body copy in default theme. For consistency, I think we should add something here.

Reply:
added